exchanges/binance/binance_test.go

```diff
@@ -2790,28 +2773,28 @@ func TestUpdateOrderExecutionLimits(t *testing.T) {
 		require.NoError(t, err, "UpdateOrderExecutionLimits should not error")
 
 		p := tests[a]
```

Comment:
🚧  This feels like a `t.Run` with a.String+"/"+p.String and removing all the f-ing